tests/manual/classiccreator.js

```diff
@@ -45,3 +45,5 @@ function destroyEditor() {
 
 document.getElementById( 'initEditor' ).addEventListener( 'click', initEditor );
 document.getElementById( 'destroyEditor' ).addEventListener( 'click', destroyEditor );
+
+initEditor();
```

Comment:
The test description is now incorrect. Change the description or remove that line.